### PR TITLE
chore: unify issue templates

### PR DESCRIPTION
**Pull request by etabakov**
utilise the new GitHub issue templates for different kind of issues

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: 'We really appreciate your effort to provide feedback. Before opening a new
+  issue, please make sure that this case is not already reported in GitHub as an
+  issue or in StackOverflow as a question.'
+
+---
+
+**Environment**
```
Comment (sis0k0) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Can we add `nativescript-angular` and `angular/*` versions here?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: 'We really appreciate your effort to provide feedback. Before opening a new
```
Comment (sis0k0) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Are the quotes (') needed?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
```
Comment (sis0k0) [inline .github/ISSUE_TEMPLATE/feature_request.md]: We can add a section for use cases.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (etabakov) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Totally, check if it's good now.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (etabakov) [reply .github/ISSUE_TEMPLATE/bug_report.md]: They were added by GitHub when editing the templates, so I presume "Yes" but haven't tried without them.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 3 above)
Comment (etabakov) [reply .github/ISSUE_TEMPLATE/feature_request.md]: This should be the first section. We can change the wording if it's not clear enough.

**Comment by SvetoslavTsenov** [comment]
test --ignore

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (sis0k0) [reply .github/ISSUE_TEMPLATE/bug_report.md]: :+1:

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (sis0k0) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Ok, I get it. It was strange to me that the second template doesn't have them.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 3 above)
Comment (sis0k0) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Ok, the first section works :)

**Pull request merged**